### PR TITLE
Panel screenshots

### PR DESCRIPTION
**Pull request by neelasha23**
----
📚 Documentation preview 📚: https://ploomber-doc--134.org.readthedocs.build/en/134/

### CODE REVIEW

File: examples/panel/chat-with-github/screenshot.png
Comment (edublancas) [inline examples/panel/chat-with-github/screenshot.png:1]: this looks fine but we haven't heard back on the resolution so let's wait a bit more

File: examples/panel/chat-with-pdf/requirements.lock.txt [hunk 1]
```diff
@@ -0,0 +1,263 @@
+aiohttp==3.8.5
+aiosignal @ file:///home/conda/feedstock_root/build_artifacts/aiosignal_1667935791922/work
```
[the remark this answers is not in the document]
Comment (neelasha23) [reply examples/panel/chat-with-pdf/requirements.lock.txt:2]: I created a new [virtual environment](https://docs.python.org/3/library/venv.html#creating-virtual-environments) and installed eveything pip

[the remark this answers is not in the document]
Comment (neelasha23) [reply examples/panel/chat-with-pdf/requirements.lock.txt]: [Fixed](https://github.com/ploomber/doc/pull/134/commits/d2ce590aa94b295adb116b06897f9f85d4996121).
i think the env didn't get activated the last time i generated the file

File: examples/panel/chat-with-github/screenshot.png
Comment (neelasha23) [reply examples/panel/chat-with-github/screenshot.png:1]: Modified image sizes and updated in webp version: https://www.platform.ploomber.io/explore/

**Review by edublancas — APPROVED** [review]

**Comment by edublancas** [comment]
Can you delete this one? 
image

**Pull request merged**